datastore/constants/rating_lock_period.go

```diff
@@ -2,38 +2,38 @@ package constants
 
 // RatingLockPeriod tells the rating slot locks what day
 // the lock should expire
-type RatingLockPeriod int16
+type RatingLockPeriodType int16
```

Comment:
```suggestion
type RatingLockPeriodDay int16
```

This just makes a bit more sense imo since all of these refer to days. Also the Godoc comment should be updated to note that this is not the real type name, since all the other types use the real name